### DISCUSSION TITLE
imodeljs-backend error and slow

### DISCUSSION THREAD

**Discussion by qingsong1900** [iTwin/itwinjs-core#1751] [Q&A]
now, when more google chrome in pc to  visit the imodeljs,and the backend log error and slow,
i want know the imodeljs backend  is  support by  single process or multi-proccess  to read the bim file from iTwin Snapshot,
when i deploy more imodeljs backend by different port,
but  when one backend process write log the sql busy and lock the bim file , and other process must wait unlock the bim file.
and all client pc must wait  a client pc .

![image](https://user-images.githubusercontent.com/85264855/123930253-273b1b80-d9c2-11eb-88c5-7eaabcdf2fb3.png)

**Comment by calebmshafer** [comment]
Not entirely sure I understand the scenario you're using here.  Do you have both a imodeljs-backend running that is used by a imodeljs-frontend application via Chrome and then also running the iTwin Synchronizer at the same time on a single iModel?
